### PR TITLE
Add cbzink/AdventOfCode2018 to PHP AoC.

### PR DESCRIPTION
**Pull request by cbzink**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Bogdanp) [inline README.md]: Looks like you added a line here. Please remove it.

File: README.md [hunk 2]
```diff
@@ -26,6 +27,7 @@ This is a collection of awesome resources related to the yearly
     * [Kotlin](#kotlin)
     * [Nim](#nim)
     * [Perl](#perl)
+    * [PHP](#php)
```
Comment (Bogdanp) [inline README.md]: `H` technically comes before `e` so please move this above `Perl` (applies to the PHP section itself, too)

File: README.md [hunk 3]
```diff
@@ -304,4 +312,4 @@ in your favourite language.*
 
 * [arknave](https://twitch.tv/arknave)
 * [josevalim](https://www.twitch.tv/josevalim)
-* [popabogdanp](https://twitch.tv/popabogdanp)
+* [popabogdanp](https://twitch.tv/popabogdanp)
```
Comment (Bogdanp) [inline README.md]: Looks like you removed the newline at the end of the file. Please add it back

**Comment by cbzink** [comment]
Sorry about that. Corrected line breaks and moved PHP above Perl in d975161.

**Pull request merged**